definitions/features/service.rb

```diff
@@ -31,7 +30,11 @@ def filtered_services(options)
     service_list = filter_services(service_list, options)
```

Comment:
As the return type of `filter_services` is modified from `array` to `hash` so could you rename this `service_list` variable to `services` or something else?